frontend/appflowy_flutter/lib/plugins/database/grid/presentation/widgets/header/mobile_grid_header.dart

```diff
@@ -41,29 +46,45 @@ class _MobileGridHeaderState extends State<MobileGridHeader> {
           fieldController: fieldController,
         )..add(const GridHeaderEvent.initial());
       },
-      child: SingleChildScrollView(
-        scrollDirection: Axis.horizontal,
-        controller: widget.anchorScrollController,
-        child: Row(
-          mainAxisSize: MainAxisSize.min,
-          children: [
-            HSpace(GridSize.leadingHeaderPadding),
-            Stack(
-              children: [
-                Positioned(top: 0, left: 24, right: 24, child: _divider()),
-                Positioned(bottom: 0, left: 0, right: 0, child: _divider()),
-                SizedBox(
-                  height: 50,
-                  child: _GridHeader(
-                    viewId: widget.viewId,
-                    fieldController: fieldController,
-                  ),
+      child: Stack(
+        children: [
+          BlocBuilder<GridHeaderBloc, GridHeaderState>(
+            builder: (context, state) {
+              return SingleChildScrollView(
+                scrollDirection: Axis.horizontal,
+                controller: widget.scrollController1,
+                child: Stack(
+                  children: [
+                    Positioned(
+                      top: 0,
+                      left: GridSize.leadingHeaderPadding + 24,
```

Comment:
We're using this for a lot of different stuff, should it still be named `leadingHeaderPadding`?

Reply:
It implies the same metric here. The amount of space between the edge and the first element in the database view header or the database view content

Reply:
Leading means it's only before, we use it all around now.

`GridSize.horizontalHeaderPadding`

Reply:
Oh I get what you mean